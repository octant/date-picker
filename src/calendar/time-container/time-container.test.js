/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import serializer from 'jest-glamor-react'

import TimeContainer from './'

expect.addSnapshotSerializer(serializer)

describe('<TimeContainer /> appearance', () => {
  test('displays the same', () => {
    const component = renderer.create(
      <TimeContainer selected='2019-07-07' />
    )

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
