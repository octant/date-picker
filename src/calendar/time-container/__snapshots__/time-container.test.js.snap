// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeContainer /> appearance displays the same 1`] = `
.css-0,
[data-css-0] {
  position: relative;
  overflow: hidden;
  padding: 0.5em;
  background-color: rgba(0, 0, 0, 0);
}

<div
  className="css-0"
/>
`;
